lib/finch.ex

```diff
@@ -32,6 +32,11 @@ defmodule Finch do
   @default_pool_count 1
 
   @pool_config_schema [
+    scheme: [
```

Comment:
Looks great! The only thing I am not sure about is calling this `:scheme`, since we already use scheme throughout the codebase to refer to `:http` or `:https`, so it could cause some confusion. Maybe we could call it `:protocol`?

Reply:
Yup. That makes way more sense.